build.gradle

```diff
@@ -1,17 +1,10 @@
 import de.itemis.mps.gradle.GitBasedVersioning
 
-buildscript {
-    repositories {
-        mavenLocal()
-        maven { url 'https://artifacts.itemis.cloud/repository/maven-mps/' }
-        mavenCentral()
-    }
-
-    dependencies {
-        classpath 'de.itemis.mps:mps-gradle-plugin:1.2.177.46341c0'
-    }
+plugins {
+    id 'de.itemis.mps.gradle.common' version '1.13.+'
```

Comment:
Where does this package name come from? How does this work?

Reply:
From here: https://github.com/mbeddr/mps-gradle-plugin/blob/v1.x/src/main/kotlin/de/itemis/mps/gradle/common.gradle.kts

It is an empty plugin that does nothing but causes Gradle to put its classes on the build script classpath.